doc/API.md

```diff
@@ -311,6 +311,7 @@ Additionally, there are a number or properties that allow configuration of diffe
 | `absolutePaths` | `false` | Use absolute paths when creating element selectors
 | `iframes`       | `true`  | Tell axe to run inside iframes
 | `elementRef`    | `false` | Return element references in addition to the target
+| `restoreScroll` | `false` | Scrolls elements back to before axe started
```

Comment:
I want to avoid that we have to recommend that you set this to false if you're using axe in a test environment. I didn't do performance tests on it, so I'm not sure if the performance impact is significant enough that we'd have to do that. But if it isn't, if you can safely run this 1000 times in your application and not have it be noticeable, than we should just do this and not have it be optional at all.

Thoughts?